langchain4j-open-ai/src/main/java/dev/langchain4j/model/openai/OpenAiTokenizer.java

```diff
@@ -31,14 +31,6 @@ public class OpenAiTokenizer implements Tokenizer {
     private final String modelName;
     private final Optional<Encoding> encoding;
 
-    /**
-     * Creates an instance of the {@code OpenAiTokenizer} for the "gpt-3.5-turbo" model.
-     * It should be suitable for all current OpenAI models, as they all use the same cl100k_base encoding.
-     */
-    public OpenAiTokenizer() {
```

Comment:
I would mark it `@Deprecated` instead of removing, otherwise it will break a lot of code